### DISCUSSION TITLE
I can only import 32 pages on the ID file

### DISCUSSION THREAD

**Discussion by Phlucena** [mike-edel/ID-MultiPageImporter#26] [Q&A]
Hello Mike! I'm having an issue with your script. I have a 300+ page document and i need to place different images to that document, one in each page, but on the script options, the maximum number of pages is 32, and i can't import 32 pictures at a time, it stops me from doing it on the 32th page. the only solution i can imagine is making multiple pdfs (from the 33th page and so on), but this would consume too much time. 
is there a fix for this problem? or the script limits the number of pages to 32?

Thanks in advance! 
Pedro

**Comment by mike-edel** [comment]
Hi Pedro,

the Issues section would probably be a better place for this but I can move our discussion there if necessary. 
Generally, there is no such limit. I have previously placed 200+ page pdf files using the script. 

To me, it sounds like there is something off with the source PDF but it's a bit hard to say with the info you provided.
Can you please share some more details?

- screenshot of the MultiPageImporter dialog
- which application was used to create the source pdf with the images
- what happens if you manually place the source pdf (does it show the number of pages in the import options dialog?)
![2022-03-29_09h54_20](https://user-images.githubusercontent.com/22517839/160562061-174df094-4c89-43d1-b0a0-b61a1a6041ae.png)
- what happens if you use a different source pdf that you have exported from InDesign (placeholder text across multiple pages)?

Expected behavior is that the script automatically fills in the number of pages in the source pdf **if it can read them**. Example screenshot when using a 201 page long empty PDF I just exported from InDesign:
![2022-03-29_09h46_20](https://user-images.githubusercontent.com/22517839/160561207-5860b7a1-ba4d-49f8-9710-3cf4e7c2ce2f.png)
